src/Symfony/Component/DependencyInjection/ChildDefinition.php

```diff
@@ -162,11 +162,21 @@ public function setDeprecated($boolean = true, $template = null)
     /**
      * {@inheritdoc}
      */
-    public function setAutowiredMethods(array $autowiredMethods)
+    public function setAutowiredCalls(array $autowiredCalls)
```

Comment:
looks good to me. And doing it now is still possible as the existing name is not yet released